L1Trigger/L1TGlobal/plugins/GenToInputProducer.cc

```diff
@@ -786,6 +842,7 @@ namespace l1t {
     srand(0);
 
     gRandom = new TRandom3();
+    mushowerRandom = new TRandom3();
```

Comment:
This should probably be deleted, or a smart pointer used to avoid a memory leak, I don't see a corresponding `delete`

Reply:
Ahh, this is all backported. Nevermind this for now